### PR TITLE
feature/about

### PR DESCRIPTION
**Pull request by Jalal999**

### CODE REVIEW

File: src/components/About/about.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        padding-left: 20px;
+        padding-right: 20px;
+
+        &__title {
+            font-size: 38px;
+            letter-spacing: 8px;
+            color: #1e2633;
+            font-family: "Axis";    
+            margin: 0;
+            margin-top: 60px;
```
Comment (Jose-Hash-Dev) [inline src/components/About/about.scss]: You can replace these 2 lines with `margin: 60px 0 0;`

File: src/components/About/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            <div className="about__texts">
+                <h1 className="about__texts__title">{title}</h1>
+                <h3 className="about__texts__subTitle">{subTitle}</h3>
+                <p className="about__texts__paragraph">{text}</p>
+            </div>
+        </div>
+    );
+};
+
+export default About;
```
Comment (Jose-Hash-Dev) [inline src/components/About/index.js]: Please use `Prettier`

**Review by Jose-Hash-Dev — CHANGES_REQUESTED** [review]

File: src/components/About/index.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import "./about.scss";
+import Structure from '../../structure.json';
+import aboutImage1 from '../../assets/img/about_img_1.png';
+import aboutImage2 from '../../assets/img/about_img_2.png';
+
+
+const About = () => {
+    const [about] = useState(Structure['about']);
+    const {title, subTitle, text} = about;
```
Comment (Jose-Hash-Dev) [inline src/components/About/index.js]: You don't have declare variables because you can access to data using `about.title/about.subTitle/about.text`

**Review by Jose-Hash-Dev — CHANGES_REQUESTED** [review]

File: src/components/About/about.scss [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            font-weight: 700;
+            font-family: "Montserrat";
+            text-transform: uppercase;
+        }
+    
+        &__paragraph {
+            font-size: 12px;
+            letter-spacing: 2px;
+            line-height: 30px;
+            color: #404040;
```
Comment (YuraLeo94) [inline src/components/About/about.scss]: Be consistent. Add color from colors variables

File: src/components/About/about.scss [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+            width: 80%;
+            height: 70%;
+            left: 20px;
+            top: 20px;
+        }
+    }
+
+    &__texts {
+        padding-left: 20px;
+        padding-right: 20px;
```
Comment (VSlobolinskyi) [inline src/components/About/about.scss]: Could be shortened to padding: 0 20px;

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]
**Note:** Please if you do any new visuals or update old ones, do not forget to attach a screenshot of the differences or what is new.

**Review by Jalal999 — COMMENTED** [review]
image
image

**Review by YuraLeo94 — APPROVED** [review]

**Review by VSlobolinskyi — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Pull request merged**